v5/pivotal/projects.go

```diff
@@ -116,3 +116,39 @@ func (service *ProjectService) Get(projectId int) (*Project, *http.Response, err
 
 	return &project, resp, err
 }
+
+func (service *ProjectService) AddStory(
```

Comment:
@dav It is true that the API is actually not very consistent with what I mentioned :-) Let me think...